### ISSUE TITLE
Mass matrix in readme

### ISSUE THREAD

**Issue by ttaa9** [dilevin/CSC2549-a6-rigid-body-contact#5]
In the expression for the mass matrix `M`, should the expression be `mI` in the bottom-right block (rather than `I`), where `m` is the total mass?

**Comment by dilevin** [comment]
yes it should be mI, this is my mistake, I think I got it right in class though, I"ll fix it now
